docs/filesystems.txt

```diff
@@ -155,5 +157,5 @@ if you need to work with revisions, as shown in the following code:
 
 .. note::
 
-   If you use a prefix the Filesystem component, you will have to handle it
+   If you specify the ``prefix`` Filesystem setting, you will have to handle it
    by yourself when using the GridFS API directly.
```

Comment:
Q: could you explain what "handle it by yourself" means? What is "it" referring to?

Reply:
Updated.